ga4gh/datamodel/reads.py

```diff
@@ -242,7 +256,7 @@ def _referenceSetInit(self, dataRepository, shouldThrowExceptions):
             elif referenceSetName != name:
                 if shouldThrowExceptions:
                     raise exceptions.MultipleReferenceSetsInReadGroupSet(
-                        self._samFilePath, name, referenceSetName)
+                        self._dataUrl, name, referenceSetName)
```

Comment:
It makes sense to me to do away with this function (`referenceSetInit`) now that we have a DB row for the readgroupset. Reading the reference set name from the file internals is an implicit assumption we probably shouldn't make. When initializing a readgroupset, providing the referenceSet that it is against would address https://github.com/ga4gh/server/issues/1050 , as well as demonstrate how the other containers would be initialized with a referenceSetId.